worlds/tunic/er_scripts.py

```diff
@@ -238,6 +255,14 @@ def pair_portals(world: "TunicWorld") -> Dict[Portal, Portal]:
                     if not portal2:
                         raise Exception(f"Could not find entrance named {p_exit} for "
                                         f"plando connections in {player_name}'s YAML.")
+                    # don't pair dead ends to dead ends, that's bad
+                    if entrance_dead_end:
+                        if isinstance(world.options.entrance_rando.value, str):
+                            raise Exception(f"Tunic ER seed group {world.options.entrance_rando.value} paired a dead "
+                                            "end to a dead end in their plando connections.")
+                        else:
+                            raise Exception(f"{player_name} paired a dead end to a dead end in their "
+                                            "plando connections.")
```

Comment:
Hmmm, another one of these duplicated code fragments. It happens either way right? So it could just sit outside the if?

Reply:
I forgot to actually reply to any of these, but I implemented pretty much all of these.